### DISCUSSION TITLE
While upscaling we reload models at every tile , can this be changed ?

### DISCUSSION THREAD

**Discussion by patientx** [ssitu/ComfyUI_UltimateSDUpscale#34] [Q&A]
I was just wondering, while upscaling with this node , with every tile, vae and "base model" gets reloaded. The loading actually takes more time than generation for me (using lcm). Is it possible to load the vae once and keep using it until node finishes its job ?

Example :

got prompt
Requested to load BaseModel
Loading 1 new model
100%|██████████████████████████████████████████████████████| 8/8 [00:11<00:00,  1.41s/it]
Requested to load TAESD
Loading 1 new model
Canva size: 1024x1536
Image size: 512x768
Scale factor: 2
Upscaling iteration 1 with scale factor 2
Tile size: 512x768
Tiles amount: 4
Grid: 2x2
Redraw enabled: False
Seams fix mode: HALF_TILE
Requested to load BaseModel
Loading 1 new model
100%|█████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.53it/s]
Requested to load TAESD
Loading 1 new model
Requested to load BaseModel
Loading 1 new model
100%|██████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.61it/s]
Requested to load TAESD
Loading 1 new model
Requested to load BaseModel
Loading 1 new model
100%|██████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.56it/s]
Requested to load TAESD
Loading 1 new model
Requested to load BaseModel
Loading 1 new model
100%|█████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.55it/s]
Requested to load TAESD
Loading 1 new model
Prompt executed in 39.69 seconds

**Comment by ssitu** [comment]
I don't think I am able to control the memory management, sorry for the late reply.

**Comment by patientx** [comment]
Thanks for replying. Keep up the good work !
